README.md

```diff
@@ -18,10 +19,31 @@ This is _work in progress_. The final config will be shared prior to launch date
 * Consensus Version: Deneb _(TBD)_
 * Merge: At Genesis (`0`)
 * Shapella Epoch: `10`
+  * Epoch time: `1694790240` (64 minutes after genesis)
 * Network ID: `17000`
 * Chain ID: `17000`
+* Genesis Hash: `0xff9006519a8ce843ac9c28549d24211420b546e12ce2d170c77a8cca7964f23d`
```

Comment:
could you add the entire genesis execution payload here pls? its easier to check if your genesis file init worked well with it

Reply:
added